backend/inspirehep/search/mappings/v7/records/hep.json

```diff
@@ -10,9 +10,11 @@
         "type": "text"
       },
       "_created": {
+        "format": "yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd||yyyy-MM||yyyy",
```

Comment:
`yyyy-MM-dd'T'HH:mm:ss.SSSSSS` that is the current format, but do we need a timestamp for that field?

Reply:
it's handled by `invenio-records`, so I'm not sure we can easily change the format here

Reply:
So we can't use other formats too?